mapbox/libjava-services/src/main/java/com/mapbox/services/api/directions/v5/MapboxDirections.java

```diff
@@ -591,6 +604,14 @@ public T setBaseUrl(String baseUrl) {
     }
 
     /**
+     * @return true it is possible to exit on the roundabout {@link #setRoundaboutExits(Boolean)}.
```

Comment:
A more descriptive doc for this given the below would be "return true if you want an additional instruction when exiting rotaries and roundabouts."